src/operations.cc

```diff
@@ -309,6 +309,20 @@ namespace sharp {
           0.0, 0.0, 0.0, 1.0));
   }
 
+  VImage Modulate(VImage image, double const brightness, double const saturation, double const hue) {
+    image = image.colourspace(VIPS_INTERPRETATION_LCH);
```

Comment:
It might be slightly more efficient to perform the colourspace conversion after the removal of alpha channel.